text/0000-lts-release-strategy.md

```diff
@@ -0,0 +1,90 @@
+
+# Motivation
+[motivation]: #motivation
+
+Administrators of production deployments find LTS releases desirable as they provide confidence that a certain software release will be maintained for an extended period of time.
+
+Specifically, for Hyperledger Fabric, administrators can expect the following of a LTS release:
+
+- Stable and well-tested code
+- API contract stability
+- 3rd digit patch releases for bug and security fixes, delivered for an extended period of time on a regular cadence (or more frequently as needed)
+- Minimal set of feature additions and other changes, reducing the risk of functional regressions and bugs
+
+Similarly, there are benefits to Hyperledger Fabric maintainers, code contributors, and the wider community:
+
+- New features and other changes can quickly be applied to the master branch, and distributed to the user community for trial, without impacting production deployments.
+- Community feedback on new features can be solicited and acted upon.
+- Bug fixes only need to be backported to a small number of designated LTS releases.
+- Extra tests (e.g. upgrade tests for non-subsequent versions) only need to be executed against a small number of designated LTS releases.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+## Release cadence
+
+Hyperledger Fabric will continue the pattern of quarterly minor releases, e.g. v2.0, v2.1, v2.2, etc.
+
+## LTS release cadence
+
+Because a new major release typically has large new features that may not yet be tried by the user community, and because SDKs may lag in support of the new release, it is not expected that a new major release will immediately be designated as a LTS release, for example, v2.0 is not a LTS release.
+
+Each major release will eventually have at least one minor release designated by the Fabric maintainers as the LTS release, e.g. v1.4 for v1.x, v2.2 for v2.x (all references to future versions are hypothetical).
+
+After a LTS release is designated, quarterly releases will continue as normal. The Fabric maintainers will decide whether a next release is another minor release, or a new major release. There is no pre-defined timing for next major version. The decision is typically based on semantic versioning considerations, such as whether API changes are needed or older capabilities need to be removed. Other considerations may also apply, for example significant upgrade steps may motivate a shift to a new major version.
+
+If a major release is not delivered for an extended period of time, the maintainers may designate a subsequent minor release as the next LTS release, for example if v3.1 is a LTS release and there is no need to increment to 4.0 for several quarters, the maintainers may decide to designate v3.4 as the next LTS release.
+

```

Comment:
So customers can set a general expectation for planning how often their businesses will want to update, is there any general timeline about how often an LTS will be released? e.g. expect around 1 LTS/per year or no more than 2 LTS per year or something like that?

Reply:
Well, the reason we're going with this approach is that we can't predict the future content and wanted to keep things flexible. Take Fabric CA for example... there is no other release planned and therefore v1.4.x will be the LTS for the foreseeable future. So I don't think we can predict timeframes at this moment.